packages/grafana-ui/src/components/Forms/RadioButtonList/RadioButtonDot.tsx

```diff
@@ -25,11 +35,15 @@ export const RadioButtonDot = ({ id, name, label, checked, disabled, description
         name={name}
         type="radio"
         checked={checked}
+        value={value}
         disabled={disabled}
         className={styles.input}
         onChange={() => onChange && onChange(id)}
       />
-      {label}
+      <div>
+        {label}
+        <div className={styles.description}>{description}</div>
```

Comment:
Do you have a screenshot of where you use this?

Reply:
![image](https://github.com/grafana/grafana/assets/6293563/540fadde-f306-4c9c-9d00-bf7350a4a8e7)

Reply:
Should the description div be optionally rendered depending on if description is provided or not?